example/storybook/stories/BrandConfig/Examples.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { Text as RNText, TextStyle } from 'react-native';
+import { Text } from '../../../../src/components/Text';
+import { View, ViewStyle } from 'react-native';
+
+interface Props {
+  content: string;
+}
+
+export function Examples({ content }: Props) {
+  return (
+    <>
+      <Container description="heading">
+        <Text variant="heading">{content}</Text>
+      </Container>
+      <Container description="subHeading">
+        <Text variant="subHeading">{content}</Text>
+      </Container>
+      <Container description="body">
+        <Text variant="body">{content}</Text>
+      </Container>
```

Comment:
I was thinking we would probably just have a few in examples, only enough to exercise the features of BrandConfigProvider. Seems like each component should also have it's own set of stories.  Almost like visual unit tests

However, if we can get to some of the more advanced storybook features, we should be able to switch up themes and styles on a global level, so that all stories are affected. Maybe to test out light vs dark themes on all components.